learn-ocaml-client.opam

```diff
@@ -21,13 +21,15 @@ depends: [
   "asak"
   "cohttp" {>= "1.0.0" & < "2.0.0"}
   "cohttp-lwt-unix" {>= "1.0.0" & < "2.0.0"}
+  "cryptokit"
   "ssl" {= "0.5.5"}
   "digestif" {>= "0.7.1"}
   "dune" {= "2.0.1"}
   "ezjsonm"
   "lwt" {>= "4.0.0"}
   "lwt_ssl"
   "ocaml" {= "4.05.0"}
+  "ocamlnet" {> "4.1"}
```

Comment:
Hm isn't `ocamlnet` a legacy library ? I am not sure it's much maintained anymore.
As far as I can see, it is only used for sending emails, did you consider alternatives ?

Reply:
Yes! It had been added just because the mirage counterpart of this library needed ocaml > 4.08, and we only had 4.05 support at this time. Thanks for reminding this need for an upgrade.